### ISSUE TITLE
Inference

### ISSUE THREAD

**Issue by ariboyarsky** [uchicago-computation-workshop/abdullah_almaatouq#43]
Thank you for presenting! Do you think there are external validity issues with the research design?

**Comment by amaatouq** [comment]
Yes.. All results (whether observational or lab/field experiments) are not environment/context-dependent.
